core/storage/certificate_assessment/gae_models_test.py

```
@@ -188,3 +190,187 @@ def test_commit_log_entry_creation_lifecycle(self) -> None:
+                attempt_id
+            )
+        )
+        self.assertEqual(updated_model.finished_at, finished_at)
+        self.assertTrue(updated_model.is_submitted)
+
+    def test_create_raises_error_when_many_id_collisions_occur(self) -> None:
+        """Ensures the ID generator raises after exhausting retries."""
+        get_by_id_swap = self.swap(
+            certificate_models.CertificateAssessmentAttemptModel,
+            'get_by_id',
+            lambda *args, **kwargs: True,
```

Ruff (0.15.20)

[warning] Unused lambda argument: `args`

(ARG005)